### PR TITLE
Sign In Screen Instant Payments Hero

### PR DESCRIPTION
**Pull request by youngkidwarrior**
Instant Payments Hero Image for Sign In page. 

Notes:
 - svg has some weird behavior because the slash cutouts baked into the sv
	- To fix this, we can make the cutouts with css, that way they won't get slice off with the sv
 - No narrow browser supportg

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/81?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `main`
  * **PR #80** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/80?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #81** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/81?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/81?utm_source=stack-comment).

File: packages/app/assets/img/react-native-svg.d.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import 'react-native-svg'
+declare module 'react-native-svg' {
```
Comment (0xBigBoss) [inline packages/app/assets/img/react-native-svg.d.ts:2]: ty

File: packages/app/components/icons/IconSendLogo.tsx [hunk 2]
```diff
@@ -1,27 +1,21 @@
...
-        fill={color ?? 'white'}
-      />
-      <Path
-        d="M32.4292 0.399994V5.98195H6.86504V8.86917C6.86504 9.44661 7.05591 9.83157 7.24042 10.0241C7.62216 10.409 8.00391 10.409 8.39201 10.409H26.9003C28.0455 10.409 29.1907 10.6015 29.9542 10.9865C30.9086 11.3714 31.4812 11.7564 32.0474 12.3338C32.6201 12.9113 33.0018 13.6812 33.3836 14.4511C33.5744 15.221 33.7653 16.1835 33.7653 17.1459V19.2632C33.7653 20.2256 33.5744 20.9955 33.3836 21.7654C33.0082 22.5353 32.6201 23.3053 32.0474 23.8827C31.4812 24.6526 30.9022 25.0376 29.9542 25.4225C29.1907 25.8075 28.0455 26 26.9003 26H0.559892V20.418H25.3669C25.7487 20.418 25.9395 20.418 26.3276 20.0331C26.5122 19.8406 26.703 19.4556 26.703 18.8782V17.5308C26.703 17.1459 26.5122 16.7609 26.3276 16.5684C25.9459 16.1835 25.755 15.991 25.3669 15.991H6.67417C5.7198 15.991 4.95632 15.991 4.00195 15.606C3.23847 15.4135 2.47498 15.0286 1.90236 14.4511C1.32974 13.8737 0.757127 13.2962 0.375382 12.5263C0.18451 11.9489 0 11.1789 0 10.2165V5.98195H6.8714V0.399994H32.4419H32.4292Z"
-        fill={color ?? 'white'}
-      />
+    <Svg width={166} height={size ?? 60} viewBox="0 0 166 60" fill="none">
```
Comment (0xBigBoss) [inline packages/app/components/icons/IconSendLogo.tsx:9]: does the size prop scale the width correctly here?

File: packages/ui/src/components/CornerTriangle.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    case 'bottomLeft':
+      return 'rotate(270deg)'
+    case 'bottomRight':
+      return 'rotate(180deg)'
+    default:
+      return 'rotate(0deg)'
+  }
+}
+
+export const CornerTriangle = (props: StackProps & CornerProp) => (
```
Comment (0xBigBoss) [inline packages/ui/src/components/CornerTriangle.tsx:20]: Very nice.

**Review by 0xBigBoss — COMMENTED** [review]
Same deal with the large images on this PR. We need to host them elsewhere and keep them under +200KB or remove them completely and instead use SVG icons.

**Review by 0xBigBoss — COMMENTED** [review]
Same deal with the large images on this PR. We need to host them elsewhere and keep them under +200KB or remove them completely and instead use SVG icons.

File: packages/app/components/icons/IconSendLogo.tsx (hunk 2 above)
Comment (youngkidwarrior) [reply packages/app/components/icons/IconSendLogo.tsx:9]: So this is how other, non-square, components are dealing with size. I don't think it actually works well. 

We will probably want to have some sort of aspect ratio multiplier determined by the original aspect of the image to properly scale svg's that aren't square

File: packages/app/components/sidebar/SignInSideBar.tsx [hunk 4]
```diff
@@ -28,9 +28,41 @@ const SignInSchema = z.object({
...
+          href={telegramSocial}
+          target="_blank"
+          borderRadius={9999}
+        />
+      </XStack>
+    </YStack>
+  </SideBar>
+)
+
+const SignInForm = () => {
```
Comment (0xBigBoss) [inline packages/app/components/sidebar/SignInSideBar.tsx:62]: Let's make this a component file underneath features/auth

File: packages/app/components/sidebar/SignInSideBar.tsx [hunk 5]
```diff
@@ -28,9 +28,41 @@ const SignInSchema = z.object({
   phone: formFields.text,
 })
 
-const SignInSideBar = ({ ...props }: YStackProps) => {
+const SignInSideBar = ({ ...props }: YStackProps) => (
```
Comment (0xBigBoss) [inline packages/app/components/sidebar/SignInSideBar.tsx:31]: this should be a layout underneath features/auth

**Comment by youngkidwarrior** [comment]
@0xBigBoss need an approval on this one as well

**Comment by youngkidwarrior** [comment]
@0xBigBoss Approval here too?

**Review by 0xBigBoss — DISMISSED** [review]

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Feb 4, 9:22 PM EST**: @youngkidwarrior started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/81).

**Comment by youngkidwarrior** [comment]
@0xBigBoss Seems like some of our Branch Protections aren't playing nice with Graphite. Causing us to re-approve a bunch

[Here are the graphite branch protection guidelines](https://graphite.dev/docs/github-configuration-guidelines)

**Comment by 0xBigBoss** [comment]
> @0xBigBoss Seems like some of our Branch Protections aren't playing nice with Graphite. Causing us to re-approve a bunch
> 
> [Here are the graphite branch protection guidelines](https://graphite.dev/docs/github-configuration-guidelines)
> 
> Let me know how you feel about these changes. If you're against any of them, I can adjust my gitflow to improve QOL

I dropped the dismiss stale review, think that should do it. with great power comes great responsibility. @youngkidwarrior

**Review by 0xBigBoss — APPROVED** [review]

**Pull request merged**